FAULT TYPE: OPENSTACK_THROW_EXCEPTION-IMAGE
TARGET COMPONENT: /cinder/api/contrib/volume_image_metadata.py
TARGET CLASS: VolumeImageMetadataController
TARGET FUNCTION DEF: show+76;12
FAULT POINT: self._add_image_metadata(context, [resp_obj.obj['volume']])
